op/model.py

```diff
@@ -314,3 +326,15 @@ def config_get(self):
 
     def is_leader(self):
         return self._run('is-leader')
+
+    def pod_spec_set(self, spec, k8s_resources):
+        tmpdir = Path(tempfile.mkdtemp('-pod-spec-set'))
+        spec_file = tmpdir / 'spec.json'
+        k8s_res_file = tmpdir / 'k8s-resources.json'
+        try:
+            spec_file.write_text(json.dumps(spec))
+            if k8s_resources:
+                k8s_res_file.write_text(json.dumps(k8s_resources))
+            self._run_no_output('pod-spec-set', '--spec', str(spec_file), '--k8s-resources', str(k8s_res_file))
```

Comment:
If k8s_resources is None its file isn't written and hopefully this will break down.

Reply:
Good catch, thanks.